### DISCUSSION TITLE
Help

### DISCUSSION THREAD

**Discussion by THEkatinamicrowave** [KadeArchive/Kade-Engine#2439] [Q&A]
Note splashes

**Comment by BrandonHatesAnime** [comment]
I know this is kinda stupid but try this https://gamebanana.com/mods/312483

**Reply by THEkatinamicrowave** [reply]
that could work, but what if I want it to happen whenever you hit a "sick" instead of every damn time you hit a note. That would be cancer in something like Tricky

**Reply by glowsoony** [reply]
i told how to do it here just go in the link and go to the comments and i told how to add tem https://gamebanana.com/requests/24055
